### PR TITLE
Add AuthorsList and Author helmet

### PR DESCRIPTION
**Pull request by YuCJ**

### CODE REVIEW

File: src/constants/link-prefix.js [hunk 1]
```diff
@@ -4,5 +4,7 @@ export const LINK_PREFIX = {
   CATEGORY: '/category/',
   TAG: '/tag/',
   TOPIC: '/topic/',
-  TOPICS: '/topics/'
+  TOPICS: '/topics/',
+  AUTHOR: '/author/',
+  AUTHORS: '/authors'
```
Comment (nickhsine) [inline src/constants/link-prefix.js]: why do we need both `AUTHOR` and `AUTHORS`?

**Comment by nickhsine** [comment]
👍 💯

**Comment by YuCJ** [comment]
Bug fix:  Add missing slash in articles author link

**Comment by YuCJ** [comment]
Add Footer to Author

**Pull request merged**